tests/dind/build.bats

```diff
@@ -212,6 +213,10 @@ function teardown() {
 
 @test "commit with removed inner image" {
 
+  if [[ $(get_platform) == "arm64" ]]; then
+	  skip "syscont-inner-img not supported on arm64."
```

Comment:
I uploaded an arm `syscont-inner-img` in the past, probably it got overwritten with an amd64-only one.

Reply:
Oh I see ... makes sense; not a big deal though.